### ISSUE TITLE
Ingresos y egresos de hospitalizados  [feature request]

### ISSUE THREAD

**Issue by ctroncoso** [MinCiencia/Datos-COVID19#1003]
Hoy en la vocería, se mostró un gráfico con la tasa de ingreso a UCI.
![image](https://user-images.githubusercontent.com/121016/121574248-11a28980-c9f4-11eb-8f14-1dc356b1df7b.png)

Es crítico para evaluar la efectividad de las vacuna, el tener información de pacientes de alta (egresos hospitalarios) anonimizados obviamente. 
La información ideal a contener, por paciente o agrupado por día, comuna de residencia y segmento de edad (rangos de 10 años) es:
Edad, 
sexo,
Estado de vacunación [0 dosis, 1 dosis, 2 dosis, esquema completo],
Severidad ( comorbilidades (0 a 3) )
Grupo GRD (existe por paciente y sería de extrema utilidad)
Estadía (en días)
Estado de egreso (Vivo, mejorado, no mejorado)

**Comment by dna33** [comment]
Esta información creo que hay que solicitarla a través de transparencia, sugiero pedir que se publique usando este github y así todos pueden utilizarla, no solo el titular de la solicitud por transparencia. Ver issue: https://github.com/MinCiencia/Datos-COVID19/issues/985

**Comment by elaval** [comment]
@dna33 una consulta: Para hacer la solicitud vía transparencia ¿Sabes si el organismo responsable de proveerles estos datos (ingresos y egresos de hospitalizados) es la Subsecretaría de Redes Asistenciales (o es la Subsecretaría de Salud / DEIS quien centraliza la provisión de datos hacia el Ministerio de Ciencia)?

**Comment by dna33** [comment]
Está distribuida esa provisión, comprende solo los datos que Minsal decide hacer públicos y en casos son entregados por DEIS, por Epi, por Redes o por Salud Pública. En este caso, los egresos hospitalarios sería la Subsecretaría de Redes Asistenciales.

**Comment by dna33** [comment]
Este fin de semana vamos a iniciar la publicación de estos datos, gracias a las solicitudes de ustedes y también a la disposición de los equipos técnicos del Ministerio de Salud de Chile.

**Comment by ctroncoso** [comment]
> Este fin de semana vamos a iniciar la publicación de estos datos, gracias a las solicitudes de ustedes y también a la disposición de los equipos técnicos del Ministerio de Salud de Chile.

Excelente noticia!  Esperemos que sea lo que se solicitó. 
A mi no me respondieron la solicitud en forma formal y realicé un reclamo. 
Lleva pendiente 7 días desde la fecha límite (23 de julio). 
https://www.portaltransparencia.cl/PortalPdT/escritorio-ciudadano?codigo=AO001T0014319

y el reclamo ingresado (27 de julio)
https://extranet.consejotransparencia.cl/Web_SCW2/Paginab/FichaCaso.aspx?ID=52017

**Comment by elaval** [comment]
> Este fin de semana vamos a iniciar la publicación de estos datos, gracias a las solicitudes de ustedes y también a la disposición de los equipos técnicos del Ministerio de Salud de Chile.

Hola @dna33 

¿Publicaron los datos el fin de semana? ¿En qué producto?

Saludos y gracias

**Comment by dna33** [comment]
Se encuentran publicados provisoriamenteaca: https://github.com/MinCiencia/Datos-COVID19/tree/master/output/producto91

**Comment by dna33** [comment]
Perdon, creo que estraba contestando a otra solicitud: Ingresos y egresos de hospitalizados [feature request] #1003

Respecto de la incidencia de infección, hospitalización y defunciones en personas sin vacuna, con 1 dosis, con 2 dosis y con pauta completa (14 días luego de completar esquema), estamos con todo lo técnico resuelto, solo falta la autorización por parte de las autoridades de Salud para lograr la publicación.

**Comment by ctroncoso** [comment]
> Perdon, creo que estraba contestando a otra solicitud: Ingresos y egresos de hospitalizados [feature request] #1003
> 
> Respecto de la incidencia de infección, hospitalización y defunciones en personas sin vacuna, con 1 dosis, con 2 dosis y con pauta completa (14 días luego de completar esquema), estamos con todo lo técnico resuelto, solo falta la autorización por parte de las autoridades de Salud para lograr la publicación.

No problem. Es solo para aclarar y no crear expectativas. 
Si gustas, podemos ver que es lo que tienen (sin los datos obviamente) para evitar que si llega la autorización, recién ahí ver que los datos no son los correctos. 
En referencia a las columnas de los datos, están todas en el formato standard CMBD del DEIS, tal cual es enviado desde los hospitales y procesados por los sistemas GRD (tengo entendido que están usando la plataforma de SIGESA). 
El único match que habría que hacer, es via RUT, con el registro de vacunación y luego anonimizar la tabla resultante, en lo posible via un hash (doble) al RUT para, sin revelar el rut de la persona, poder analizar reingresos/infecciones.

**Comment by dna33** [comment]
No, los datos que se encuentran autorizados par a publicación son agregados por semana epidemiológica y a nivel nacional. Totales e incidencias en base a la población INE.

**Comment by ctroncoso** [comment]
> No, los datos que se encuentran autorizados par a publicación son agregados por semana epidemiológica y a nivel nacional. Totales e incidencias en base a la población INE.

Impeque. A esperar la respuesta de los egresos del minsal entonces. Gracias por todo el trabajo!

**Comment by elaval** [comment]
Hola Demián,

Hoy en el reporte diario ya anunciaron que estos datos estarían disponibles
en el repositorio de datos :-)

¿Cúando estará en producción?

Gracias por todo el esfuerzo y buen trabajo

**Comment by MinCiencia** [comment]
Asi es, estamos en los últimos detalles y autorizaciones. Estimo podamos publicarlos mañana antes de medio día.

**Comment by MinCiencia** [comment]
Publicado:

https://github.com/MinCiencia/Datos-COVID19/tree/master/output/producto89
https://github.com/MinCiencia/Datos-COVID19/tree/master/output/producto90

**Comment by alonsosilvaallende** [comment]
Muchísimas gracias!

**Comment by elaval** [comment]
Excelente!  Gracias
